README.md

```diff
@@ -1,4 +1,20 @@
 # TFExplorer
+[![Build Status](https://travis-ci.org/hardlyhuman/TFExplorer.svg?branch=master)](https://travis-ci.org/hardlyhuman/TFExplorer)
+
+
+<b>TFExplorer enables developers to quickly find out the relevant TensorFlow API's for their needs & requirements.
+It basically deals with different tools for TensorFlow API's .
+This repository is for Backend Development part &  Django is the tech stack we are using for it.</b>
+
+<b>Those who wish to contribute may visit this url:  https://github.com/hardlyhuman/TFExplorer/blob/master/.github/contributing.md for contribution guidelines.</b>
+
+<b>Those who are interested may join our telegram channel https://t.me/joinchat/ME1g6xj6Y9vLcQhjiMSNyw  for futher communication purposes.</b>
```

Comment:
There is no need for this line.
Just put a telegram badge near the Travis badge